lib/modules/manager/npm/readme.md

```diff
@@ -7,6 +7,9 @@ The following `depTypes` are currently supported by the npm manager :
 - `engines` : Renovate will update any `node`, `npm` and `yarn` version specified under `engines`.
 - `volta` : Renovate will update any `node`, `npm`, `pnpm` and `yarn` version specified under `volta`.
 - `packageManager`
+- `overrides`
+- `resolutions`
+- `pnpm.overrides`
```

Comment:
When looking at the documentation checkbox for this PR, I noticed that the trifecta of `overrides`, `resolutions` and `pnpm.overrides` was missing from the `depTypes`. I added it here for completeness, but I can split this to another PR if needed.